crd-generator/src/main/java/io/strimzi/crdgenerator/CrdGenerator.java

```diff
@@ -63,7 +63,7 @@
  * <p>The tool works by recursing through class properties (in the JavaBeans sense)
  * and the types of those properties, guided by the annotations.</p>
  *
- * <h3>Annotations</h3>
+ * <h2>Annotations</h2>
```

Comment:
The build was failing for me locally with JDK 14 due to more strict ordering for headers in Javadoc.